ts/utils/cookieManager.ts

```diff
@@ -1,14 +1,15 @@
 import CookieManager, { Cookie } from "@react-native-cookies/cookies";
 
-export const ioClearCookie = (cb: () => void) => {
-  CookieManager.clearAll().catch(cb);
-};
+export const ioClearCookie = (cb: () => void) =>
+  void CookieManager.clearAll().catch(cb);
```

Comment:
To get the same style, could it make sense to rename this function by removing the `io` prefix?

Reply:
Sounds good